test/regression/runSingleTest.ts

```diff
@@ -3,22 +3,88 @@ import {By, snap, Snappit, IConfig, WebDriver, WebElement} from "snappit-visual-
 
 import * as util from "../common/util";
 
-export async function regressionTest(browser: string, component: string) {
-    
+
+export async function regressionTest(browser: string, component: string, buildIdentifier: string) {
+    interface IResolutions {
+        [key:string]: string
+        firefox: string;
+        chrome: string;
+        safari: string;
+        "internet explorer": string;
+        MicrosoftEdge: string;
+    }
+
+    const screenResolution = "1600x1200";
+
+    interface IPlatforms {
+        [key: string]: string
+        firefox: string;
+        chrome: string;
+        safari: string;
+        "internet explorer": string;
+        MicrosoftEdge: string;
+    }
+
+    const MAC = "macOS 10.13";
+    const platforms: IPlatforms = {
+        firefox: MAC,
+        chrome: MAC,
+        safari: MAC,
+        "internet explorer": "Windows 10",
+        MicrosoftEdge: "Windows 10"
+    };
+    const platform = platforms[browser];
+
     const config: IConfig = {
-        browser: browser
+        browser: browser,
+        screenshotsDir: "artifacts/regressionScreenshots",
```

Comment:
Do we need this if we're not taking screenshots?

Reply:
We don't need this for dom-diff. We need it only for visreg